main.go

```diff
@@ -46,58 +27,44 @@ func main() {
 
 func runMain() error {
 	// Check if we're in the root of a Git repo
-	currentWorkingDirpath, err := os.Getwd()
+	currentWorkingDirectory, err := os.Getwd()
```

Comment:
This `Dirpath` name is actually significant; check this guy: https://github.com/kurtosis-tech/kurtosis-internal-docs/blob/master/general-style-guide.md#when-naming-filedirectory-path-variables-